Examples/MAX78002/CSI2/main.c

```diff
@@ -184,29 +230,17 @@ int main(void)
     printf("python grab_image.py [COM#] [baudrate]\n");
```

Comment:
Forgot to comment on this. The printf statements should be updated since grab_image.py was removed.